app/uk/gov/hmrc/incorporatedentityidentification/config/AppConfig.scala

```diff
@@ -36,7 +36,7 @@ class AppConfig @Inject()(config: Configuration, servicesConfig: ServicesConfig)
 
   def getRegisterWithMultipleIdentifiersUrl: String = {
     val baseUrl = if (isEnabled(DesStub)) desStubBaseUrl else desBaseUrl
-    s"$baseUrl/cross-regime/register/VATC"
+    s"$baseUrl/cross-regime/register/GRS?grsRegime=valueCallingServiceProvides"
```

Comment:
This needs to be the value that is provided in the journey config, e.g. PPT or VATC. It will probably have to be passed from the controller to the connector and then passed here